pyproject.toml

```diff
@@ -48,3 +51,19 @@ disable = [
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.mypy]
+exclude = ".venv"
+strict = true
```

Comment:
Could you explain these settings for me (or point me to a good reference)?

Reply:
Here's the docs for [`--strict`](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-strict).